server/conf/routes

```diff
@@ -95,6 +95,7 @@ GET     /admin/programs/:programId/applications/all                       contro
 GET     /admin/programs/:programId/applications/singleVersion             controllers.admin.AdminApplicationController.downloadSingleVersion(request: Request, programId: Long)
 GET     /admin/programs/:programId/applications/:applicationId            controllers.admin.AdminApplicationController.show(request: Request, programId: Long, applicationId: Long)
 POST     /admin/programs/:programId/applications/:applicationId/updateStatus            controllers.admin.AdminApplicationController.updateStatus(request: Request, programId: Long, applicationId: Long)
+POST     /admin/programs/:programId/applications/:applicationId/editNote            controllers.admin.AdminApplicationController.editNote(request: Request, programId: Long, applicationId: Long)
```

Comment:
? should we use update like with updateStatus?  edit feels better in english but I wonder if we are meant to adhere to a smaller set of RESTful verbs.

Reply:
Converted to updateNote for parity. IMO, paths aren't what give RESTfulness, it's the associated HTTP verb. A truly "RESTful" setup here would be a `POST` to `/admin/programs/:programId/applications/:applicationId/notes`. We don't end up following that convention elsewhere, so likely not worth converting to it here.

Reply:
Ah true true.  I think I was miss using RESTful to get at having common/readable terms in our public controller methods.